src/main/java/ch/cern/eam/wshub/core/services/workorders/impl/ChecklistServiceImpl.java

```diff
@@ -108,9 +108,13 @@ private void getResponsibilityDescriptions(InforContext context, WorkOrderActivi
 		String reviewerQualification = workOrderActivityCheckList.getReviewerQualification();
 		String performer1Qualification = workOrderActivityCheckList.getPerformer1Qualification();
 		String performer2Qualification = workOrderActivityCheckList.getPerformer2Qualification();
-		List<String> qualifications = new ArrayList<>();
-		for(UserQualification qualification : workOrderActivityCheckList.getUserQualifications())
-			qualifications.add(qualification.getUserDefinedCode());
+		List<String> qualifications = null;
+
+		if(workOrderActivityCheckList.getUserQualifications() != null){
+			qualifications = workOrderActivityCheckList.getUserQualifications().stream()
+					.map(qualification -> qualification.getUserDefinedCode())
```

Comment:
As with the `.finally(() => { ... })` in the other PR, it is arguably better to specify the function directly: `.map(WorkOrderActivityCheckListDefaultResult::getUserDefinedCode)`